core/build.sbt

```diff
@@ -4,7 +4,7 @@ resolvers ++= Seq(
 
 libraryDependencies ++= {
   Seq(
-    "co.fs2"            %% "fs2-core" % "0.10.0-M10",
+    "co.fs2"            %% "fs2-core" % "0.10.0-M11",
```

Comment:
Unsure whether this is bincompat with 0.10.0-M10.  We'll want to think about versioning.  The end goal here is to get on 0.10.0 when it's out.